### PR TITLE
Change #,# delimiter in web apps multi-select case search fields

### PR DESCRIPTION
**Pull request by orangejenny**
## Product Description
https://dimagi-dev.atlassian.net/browse/USH-2383

## Technical Summary
Changes usage of `#,#` delimiter to standard CSV format: space delimited, with multi-word values in quotes and quotes escaped by backslashes when needed.

Marking don't merge because this will depend on a formplayer PR.

## Feature Flag
Case claim

### Safety story
Reasonable test coverage, will probably ask QA to take a pass.

### Automated test coverage

In PR

### QA Plan

Probably

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-        dateFormat = "YYYY-MM-DD";
-    var selectDelimiter = "#,#"; // Formplayer also uses this
-    var Const = hqImport("cloudcare/js/form_entry/const"),
+    let kissmetrics = hqImport("analytix/js/kissmetrix"),
+        FormplayerFrontend = hqImport("cloudcare/js/formplayer/app"),
+        separator = " to ",
+        dateFormat = "YYYY-MM-DD",
+        Const = hqImport("cloudcare/js/form_entry/const"),
+        Util = hqImport("cloudcare/js/formplayer/utils/util"),
         Utils = hqImport("cloudcare/js/form_entry/utils"),
```
Comment (snopoke) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js:13]: nit: having `Util` and `Utils` in the same namespace is confusing - consider renaming one?

**Review by snopoke — APPROVED** [review]

**Review by stephherbers — APPROVED** [review]

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js:13]: The best part is that `util` is in a `utils` folder. Yeah, I'll take a look in another PR.

Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js:13]: https://github.com/dimagi/commcare-hq/pull/32148

**Comment by orangejenny** [comment]
Updated to remove the setting and add a formplayer flag that indicates the new behavior, so that this can be deployed independently of formplayer. Still working through some issues with merging the util/utils rename.

**Comment by orangejenny** [comment]
The remaining lint error looks like a possible actual issue, although this PR wouldn't have introduced it. Tests for sticky values would be excellent.

**Review by snopoke — APPROVED** [review]

**Comment by orangejenny** [comment]
Closing, see discussion on ticket.

**Pull request closed without merging**